packages/react-native-editor/__device-tests__/gutenberg-editor-drag-and-drop.test.js

```diff
@@ -12,8 +12,9 @@ import {
 } from './helpers/utils';
 import testData from './helpers/test-data';
 
-// Used to skip some tests on iOS
-const onlyOnAndroid = isAndroid() ? it : it.skip;
+// Tests associated with this const are temporarily off for both platforms due to failures.
+// They should be enabled for Android-only when a fix is in place.
+const onlyOnAndroid = it.skip;
```

Comment:
@SiobhyB @derekblank I feel it can be a bit misleading that we keep calling the constant `onlyOnAndroid` when we are skipping the test on both platforms 😅. Maybe we could directly use `it.skip` on those tests and add this comment as a reference that we'll re-enable them in the future when we provide a fix.